src/main/scala/higherkindness/rules_scala/workers/zinc/common/ColorMessage.scala

```diff
@@ -0,0 +1,28 @@
+package higherkindness.rules_scala
+package workers.zinc.common
+
+import xsbti.Problem
+import Console.{RED => CR, YELLOW => CY, GREEN => CG, RESET => CRESET}
+
+class ColorMessage() {
+  def CProblem(problem: Problem, color: String): String = {
+    val CC = color match {
```

Comment:
Pattern matching on a string seems like a bad idea. How about we break this out into separate methods according to color?

Usage then looks something like:
```
Color.formatYellow("my message")
Color.formatRed("a red message")
```

Reply:
I personally think `formatWarning` makes more sense.